WordPress/Classes/Models/DiffAbstractValue+Attributes.swift

```diff
@@ -4,11 +4,11 @@ extension DiffAbstractValue {
         case .add:
             return [.backgroundColor: WPStyleGuide.extraLightBlue(),
```

Comment:
Do we need to do these too?

Reply:
I had some trouble getting that screen to appear correctly, so I skipped it.